ci/do-audit.sh

```diff
@@ -113,6 +113,17 @@ cargo_audit_ignores=(
   # Dependency tree:
   # ring 0.17.3
   --ignore RUSTSEC-2025-0009
+  
```

Comment:
CI says you have some extra whitespace on line 116
https://buildkite.com/anza/agave/builds/21794#0196026f-3cd1-4562-9aa7-c576ba8760d3